test-configs.yaml

```diff
@@ -1715,6 +1720,12 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: imx8mq-zii-ultra-zest
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest
```

Comment:
Please drop `kselftest` for now as it's not quite usable. We're still fixing issues with the builds, so tests are not running correctly yet.

About `baseline-nfs`, well it doesn't seem to be working so might be better to leave it out initially:
https://hekla.openlab.pengutronix.de/scheduler/job/387288

Reply:
baseline-nfs should actually work.
But someshow for the used defconfig the eth0 device does not seem to show up:

[    1.671854] IP-Config: Failed to open eth0
[    1.675980] IP-Config: Device `eth0' not found